tests/test_switch.py

```
@@ -2178,6 +2178,129 @@ def fake_add_entities(entities: Iterable[Any], _update_before_add: bool = False)
     assert calls.get("len") == 1
 
 
+@pytest.mark.asyncio
+async def test_async_setup_entry_creates_vpn_switch_when_name_is_missing(
+    coordinator: MagicMock,
+    ph_hass: Any,
```

Ruff (0.15.18)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `ph_hass`

(ANN401)